### ISSUE TITLE
std for Scala developers

### ISSUE THREAD

**Issue by EECOLOR** [paulp/psp-std#34]
I noticed that I am sometimes looking for the right way to translate things. Is it a good idea to open a wikipage for things like the following?

- Implicit macro's might silently fail when they introduce reference to stuff from the standard library. To figure out the dependencies, call the implicit method directly.
- `SetView` is the equivalent of `Set` (is it?)
- What collection to use where I previously used `Seq`?

Maybe I missed some documentation somewhere.

**Comment by paulp** [comment]
Nope, you haven't missed any documentation. I'll see what I can do. Using the wiki sounds great.

**Comment by paulp** [comment]
By the way I'm still hoping to achieve a huge simplification in the views. The problem is that bits of what currently works are spread across at least two overlapping approaches to all the implicits, and they're not proving that easy to disentangle.

**Comment by EECOLOR** [comment]
That's fair. I'll use this thread for stuff that could use a bit of clarification. No need to respond to it, but when you feel like answering some (for you) more easy questions, you can do it here. Eventually we can use the raw contents of the issues to create documentation / tests / specification.

**Comment by EECOLOR** [comment]
The types in the API could use some explanation. For example:

What is the difference between `ExSet` and `InSet`, when to use which?

The same goes for `Each`, `Indexed`, `Direct` and `Linear`

**Comment by dwijnand** [comment]
NB You can't pull request against GitHub's wikis, so there's that to bear in mind.. perhaps markdown in the repo as is works well enough.

**Comment by jeantil** [comment]
@dwijnand 
Since this issue and the comments, github has implemented a way to publish documentation from master : https://github.com/blog/2233-publish-your-project-documentation-with-github-pages 

* Create a /docs/index.md file on your repository's master branch.
* Add your content and any necessary Jekyll front matter, then commit your changes.

this could replace the wiki and allow PRs ... it's not quite as easy to edit but makes it easier to contribute

**Comment by paulp** [comment]
@jeantil Thanks, I like this idea.

**Comment by EECOLOR** [comment]
@jeantil Awesome, I missed that one, thanks for the link.
